resim/metrics/python/metrics.py

```diff
@@ -216,13 +209,47 @@ def unpack_common_fields(cls, msg: metrics_pb2.Metric) -> Metric[Any]:
         else:
             unpacked.order = None
 
+        if len(msg.tags) > 0:
+            unpacked.kv_tags = []
+            for tag in msg.tags:
+                unpacked.kv_tags.append(Tag.unpack(tag))
+        else:
+            unpacked.kv_tags = None
+
         if msg.HasField("event_metric"):
             unpacked.event_metric = msg.event_metric
         else:
             unpacked.event_metric = None
 
         return unpacked
 
+    @classmethod
```

Comment:
nice!

Reply:
Ah yes, refactored this out as the linter was complaining `unpack_common_fields` had too many statements